apps/web/pages/api/v1/teams/[teamId]/members/[userId]/index.ts

```diff
@@ -24,6 +24,35 @@ export default async function handle(req: NextApiRequest, res: NextApiResponse)
     return res.status(400).json({ message: "Missing userId" });
   }
 
+  const hasOwnerOrAdminAccess = await isAdminOrOwner(currentUser, teamId);
```

Comment:
This database call is only used in the PATCH method, so it's better to move it to the `if (req.method === "PATCH") {` block to avoid unnecessary database calls when an invite should be deleted.